src/database.ts

```diff
@@ -128,4 +128,9 @@ export class Database {
       comment: options?.comment,
     });
   }
+
+  // deno-lint-ignore no-explicit-any
+  runCommand<T = any>(body: Document): Promise<T> {
```

Comment:
Hey Thanks for the PR! Can you change that the a type union of `Document | string` as per spec?

Reply:
Do you want it to be compatible with [mongosh method](https://www.mongodb.com/docs/manual/reference/method/db.runCommand/)? If so, do you want it to `JSON.parse` string in `db.runCommand`, or to change the signature in every other lib place and `JSON.parse` it in `WireProtocol`?

In contrast, this lib is incompatible with mongosh in many other ways. However, even official mongodb driver doesn't support this "compatibility" and particularly [db.command](https://github.com/mongodb/node-mongodb-native/blob/fcff6feed0e569137071b0220f66730d60b1b059/src/db.ts#L2) with string docs.

Reply:
ahh okay :) im sorry that the response took so long